### PR TITLE
Milestone 1 - setup controllers

### PR DESCRIPTION
**Pull request by sneeu-leeu**
In this milestone I:
- created controllers for posts and users
- set up routes to posts as homepage and users and posts as index and show
- created views for the pages as shown in the wireframe

Please review my project and let me know how I can improve!

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (henatan99) [inline app/controllers/posts_controller.rb]: - [x] PostsController is required to have `index` and `show` actions for this milestone. You may not need to define it but declare it. Kindly add them. :pray:

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (henatan99) [inline app/controllers/users_controller.rb]: - [x] UsersController is required to have `index` and `show` actions for this milestone. You may not need to define it but declare it. Kindly add them. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sneeu-leeu** [comment]
Done! Thanks for the feedback @henatan99

**Comment by Davidosky007** [comment]
## Hi @sneeu-leeu 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 


nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**